js/index.js

```diff
@@ -1,7 +1,108 @@
 // Iteration 1: Names and Input
-
+console.log("I'm ready!");
+const hacker1 = 'Friso';
+console.log('The driver name is '+hacker1)
+const hacker2 = 'Boris';
+console.log('The navigator name is '+hacker2);
 
 // Iteration 2: Conditionals
 
+if ( hacker1.length > hacker2.length ) {
+console.log(`The driver has the longest name, it has ${hacker1.length}characters.`)
+}
+else if (hacker2.length > hacker1.length) {
+console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+
+}
+else if (hacker1.length === hacker2.length) {
+console.log(`Wow, you both have equally long names, ${hacker1.length} characters!.`)
+
+}
+
+// Iteration 3: Loops
+
+hacker1Uppercase = hacker1.toUpperCase()
+nameWithSpaces = '';
+for (let i = 0; i < hacker1Uppercase.length; i++) {
+console.log(hacker1Uppercase[i] );
+nameWithSpaces += `${hacker1Uppercase[i]} `;
+nameReverseOrder = '';
```

Comment:
If you're not using it inside the loop, It's better to define nameReverseOrder in the global scope.